modules/form-field/css/lib/form-field.scss

```diff
@@ -1,13 +1,13 @@
 // Fields
 @mixin wdc-form-field-wrapper() {
-  margin-bottom: $wdc-spacing-s;
+  margin-bottom: $wdc-spacing-m;
   position: relative;
 
   .wdc-form-field {
     position: relative;
-    min-height: $wdc-spacing-l;
     display: flex;
     flex-direction: column;
+    justify-content: center;
```

Comment:
What was this needed for? Do we want all fields to be left aligned?

Reply:
It's a vertical alignment (`flex-direction: column`)